### PR TITLE
Color workout card

### PR DESCRIPTION
**Pull request by BerePendzialek**
Hola! Here is the PR for my 3. Story. The color of the workout sections is set according to the workout intensity (FTP). I look forward to your suggestions. Best, Berenice

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import styled from 'styled-components/macro'
 
 export default styled.button`
-  border: 1px solid orange;
-  width: 30%;
-  padding: 8px;
+  border: 0.5px solid grey;
```
Comment (macfrei) [inline src/components/Button/Button.js]: Why don't you use your color variable here?
```suggestion
  border: 0.5px solid var(--color-grey);
```

File: src/services/colorSection.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export { colorSection, colorSectionInterval }
```
Comment (macfrei) [inline src/services/colorSection.js]: `colorSection` is an interesting name. Maybe `selectColor` and `selectColorByInterval`? I couldn't understand what was meant by `colorSection`.

**Review by macfrei — COMMENTED** [review]
Well done! I added a few comments you might want to consider. What is missing are some tests for your color functions 🤖 🚀

File: src/services/colorSection.js (hunk 2 above)
Comment (BerePendzialek) [reply src/services/colorSection.js]: Thanks! I changed it accordingly

**Pull request merged**